src/number-util/number-util.ts

```diff
@@ -0,0 +1,23 @@
+export namespace NumberUtil {
+  export function intValueOf(numStr: string): number {
+    const result = isNumeric(numStr) ? Number(numStr) : NaN;
+
+    if (Number.isNaN(result) || !Number.isInteger(result)) {
+      throw new Error(`unable to convert "${numStr}" to integer type`);
+    }
+    return result;
```

Comment:
Number.isNaN()를 불필요하게 두번 하고 있습니다. 1
```suggestion
    const result = Number(numStr);

    if (!numStr || !Number.isInteger(result)) {
      throw new Error(`unable to convert "${numStr}" to integer type`);
    }

    return result;
```

Reply:
아 제가 너무 꼬아서 생각했나봅니다 😂 간단하게 할 수 있었던 것을.. 감사합니다! 제안대로 수정했습니다